plan/logical_plan_test.go

```diff
@@ -1459,23 +1459,23 @@ func (s *testPlanSuite) TestUniqueKeyInfo(c *C) {
 		{
 			sql: "select a, sum(e) from t group by a",
 			ans: map[string][][]string{
-				"TableScan_1":   {{"test.t.a"}},
+				"TableScan_1":   {{"test.t.f"}, {"test.t.g"}, {"test.t.f", "test.t.g"}, {"test.t.a"}},
```

Comment:
The position of calling eliminateAggregation is chagned in optimizer.go. Now it's before PrunColumns. So key is more than before.